FirebaseInAppMessaging/Sources/Data/FIRIAMFetchResponseParser.m

```diff
@@ -335,14 +335,20 @@ - (FIRIAMMessageDefinition *)convertToMessageDefinitionWithMessageDict:(NSDictio
                                                messageName:messageName
                                                contentData:msgData
                                            renderingEffect:renderEffect];
-
+    NSDictionary *dataBundle = nil;
```

Comment:
what are these changes?

Reply:
I ended up pull in changes from master to add 'GoogleDataTransportCCTSupport', '~> 1.3', but I forgot to also pull in the latest changes into the feature branch. This is all resolved.